Two more community call blog posts, last from 2019 (#72)
publiccodenet/blog: _posts/2019-12-13-notes-from-community-call-5-december-2019.md

---
title: "Notes from community call - 5 December 2019"
date: 2019-12-13
author: Jan Ainali
type: blogpost

> excerpt: News from Mexico City, and when to apply the Standard

Reviewer: ```suggestion
excerpt: An update from Mexico City, and when to apply the Standard
```

> excerpt: We discussed how the Standard can be applied and building communities.

Reviewer: ```suggestion
excerpt: News from Mexico City, and when to apply the Standard
```
